_docs/reference/release_notes.md

```diff
@@ -25,31 +25,128 @@ release notes.
 We recommend always using "tagged" versions of StorageOS rather than "latest",
 and to perform upgrades only after reading the release notes.
 
-## Known Issues
+## Upgrades
 
-- Once a 0.8.0 cluster has been established, it is currently not possible to add
-  or remove members.  We expect this functionality to come in 0.8.1, and welcome
-  feedback on how this should behave.
-- `storageos volume mount <vol> <mountpoint>` sometimes hangs on Managed Plugin
-  installs.  Volumes mount into containers correctly using Docker.
-- Docker can only access volumes created in the `default` namespace.
-- Clients mounting volumes from RHEL7/CentOS 7 will experience degraded
-  performance due to the absence of the [nbd kernel module]({%link _docs/install/prerequisites/nbd.md %}) on those platforms.
+Upgrades can be performed by restarting to StorageOS node container with the
+new version, or by performing `docker plugin upgrade` on Docker Managed Plugin
+installs.
 
-## Upgrades
+Before a node is upgraded, applications local to the node that mount StorageOS
+volumes should be migrated to other cluster nodes or their data volumes will be
+unavailable while the StorageOS container restarts.
+
+Where there are special considerations they are described below.
+
+### 0.8.x -> 0.9.x
+
+Start-up sripts should be updated to use the new cluster discovery syntax (See:
+https://docs.storageos.com/docs/install/prerequisites/clusterdiscovery)
+
+Do not mix a cluster with 0.8.x and 0.9.x versions as port numbers have changed.
+This may cause cluster instability while nodes are being upgraded.
+
+### 0.7.x -> 0.8.x
 
 Due to the nature the KV Store change there is no upgrade method from 0.7.x to
-0.8.0+.  Our recommendation is to create a new cluster, paying attention to the
+0.8.x+.  Our recommendation is to create a new cluster, paying attention to the
 new parameters (`CLUSTER_ID` and `INITIAL_CLUSTER`)
```

Comment:
The documentation for `INITIAL_CLUSTER` isn't up any more. I would remove this path entirely - I doubt there are still 0.7 clusters around from before August.

Reply:
Since this is in the 0.7.x -> 0.8.x upgrade section I'd like to leave it in, but I'll add a note to explain it's now deprecated.